templates/app/models/sequelize-model.hbs

```diff
@@ -28,18 +28,18 @@ module.exports = (sequelize, DataTypes) => {
     {{#if reference.isBelongsToMany}}
       through: '{{reference.through}}',
       foreignKey: '{{reference.foreignKey}}',
-      otherKey: '{{reference.otherKey}}',{{#if reference.as}}
-      as: '{{reference.as}}',{{/if}}
+      otherKey: '{{reference.otherKey}}',
     {{else}}
       foreignKey: {
         name: '{{reference.foreignKeyName}}',
         field: '{{reference.foreignKey}}',
-      },{{#if reference.targetKey }}
-      target: {
```

Comment:
This option is not handled by Sequelize see the [documlentation](https://sequelize.org/master/class/lib/model.js~Model.html#static-method-belongsTo) about this.